### PR TITLE
Feature/sc 32768/library module route updates for clark refactor

### PR DESCRIPTION
**Pull request by Cwagne17**
This story fixes the library service routes and broken logic that didn't allow for pagination and simplified some roundabout legacy logic of getting library _id by using the learningObject that exists on the document etc. etc..

### CODE REVIEW

File: src/app/cube/library/library.component.ts [hunk 1]
```diff
@@ -124,17 +124,20 @@ export class LibraryComponent implements OnInit, OnDestroy {
...
+
+      this.libraryItems.map(async (libraryItem: LibraryItem) => {
+        const ratings = await this.getRatings(libraryItem.learningObject);
         if (ratings) {
           libraryItem['avgRating'] = ratings.avgValue;
         }
       });
       this.loading = false;
     } catch (e) {
+      console.log(e);
```
Comment (johngonz0) [inline src/app/cube/library/library.component.ts:140]: is this an intentional console.log

**Review by johngonz0 — APPROVED** [review]
looks good just one question

File: src/app/cube/library/library.component.ts (hunk 1 above)
Comment (pzalep1) [reply src/app/cube/library/library.component.ts:140]: The library does not handle errors.

**Review by pzalep1 — APPROVED** [review]

**Review by stef359 — APPROVED** [review]

File: src/app/cube/library/library.component.ts (hunk 1 above)
Comment (Cwagne17) [reply src/app/cube/library/library.component.ts:140]: Yes because otherwise we wouldn't know what the issue was in the client.

**Pull request merged**